src/apps/bridge/sensor_drivers/softSensor.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "FreeRTOS.h"
+#include "abstractSensor.h"
+#include "avgSampler.h"
+#include "sensorController.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#define SOFT_NUM_SAMPLE_MEMBERS 1
+
+typedef struct soft_aggregations_s {
+  double temp_mean_deg_c;
+  uint32_t reading_count;
```

Comment:
I autopilot-added reading_count to the SOFT sensors, but based on discussions this has been punted till later. 

I can leave this in or remove it. Either way it is not being sent out, but it is being logged in the AGG log (although the  IND/AGG logs will change in another PR too, so I can remove it in the PR if that makes sense to)